### ISSUE TITLE
During Runner Group Update selected repositories set to empty list

### ISSUE THREAD

**Issue by Vladyslav-Miletskyi** [integrations/terraform-provider-github#967]
During Group creation everything is OK, but update sets allowed repositories to None.

### Terraform Version
Independent of terraform version.
Tested with 4.17.0 provider

### Affected Resource(s)
- github_actions_runner_group

### Terraform Configuration Files
```hcl
resource "github_actions_runner_group" "project" {
  name                    = var.group_name
  visibility              = "selected"
  selected_repository_ids = values(data.github_repository.this)[*].repo_id
}
```

### Expected Behavior
When updating the list of selected repositories terraform updates list of these repos on the GitHub side. 

### Actual Behavior
The selected repositories list is set to empty after update.

### Steps to Reproduce
1. Create New Runners Group with some selected repos
2. Run apply
3. Update list of selected repos in terraform
4. Run apply

**Comment by Vladyslav-Miletskyi** [comment]
The problem might be caused by missing update functionality for selected_repos field

**Comment by jcudit** [comment]
Any chance this was fixed by 3a19681?

**Comment by Vladyslav-Miletskyi** [comment]
@jcudit yeah should be fixed
